<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1452</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="BoxN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1451</width>
      <height>711</height>
     </rect>
    </property>
    <property name="title">
     <string>Neuronas...</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Insertar Inicio</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingresar </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoN">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoN_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoN_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoN_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoN_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoN_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoN_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>240</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Insertar Final</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>280</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Neuronas Actuales.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltaje</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Posicion de X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Posicion de Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Green</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="TextEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="EliminarInicioB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar Inicio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Open_File">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>280</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EliminarFinalB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar Final</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="TextoD">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>100</y>
       <width>191</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Directorio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Open_File_2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>280</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Recuperar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>420</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="datoN_8">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>390</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>611</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>10</y>
       <width>661</width>
       <height>681</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>670</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Dibujar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>590</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ordenar</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>611</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1452</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
